### PR TITLE
Add `role_name` parameter to `auth.token.create_orphan`

### PR DESCRIPTION
**Pull request by briantist**
#869 added the `create_orphan` method, but we were missing `role_name`.

This also fixes some headings in the changelog.

### CODE REVIEW

File: hvac/api/auth_methods/token.py [hunk 1]
```diff
@@ -156,6 +157,8 @@ def create_orphan(
             The ID provided may not contain a `.` character. Otherwise, the
             token ID is a randomly generated value.
         :type id: str
+        :param role_name: he name of the token role.
```
Comment (colin-pm) [inline hvac/api/auth_methods/token.py:160]: nitpick: typo

```suggestion
        :param role_name: The name of the token role.
```

**Review by colin-pm — CHANGES_REQUESTED** [review]
One small doc issue in token.py, but otherwise lgtm!

Thanks for fixing the docs!

**Review by colin-pm — CHANGES_REQUESTED** [review]
If you're planning on immediately merging this into master to create the 1.0.1 patch release, you'll need to bump the version in pyproject.toml. The easiest way to do it is with this command.

```bash
poetry version patch
git add pyproject.toml
...
```

**Comment by briantist** [comment]
> If you're planning on immediately merging this into master to create the 1.0.1 patch release

Thanks! Per our offline discussion, let's do the release in a separate PR, and we'll ensure #894 is included in the release

**Review by colin-pm — APPROVED** [review]
LGTM

**Pull request merged**